src/TributeRange.js

```diff
@@ -144,7 +144,7 @@ class TributeRange {
                     : '\xA0'
                 text += textSuffix
                 this.pasteHtml(text, info.mentionPosition,
-                    info.mentionPosition + info.mentionText.length + !this.tribute.autocompleteMode)
+                    info.mentionPosition + info.mentionText.length + info.mentionTriggerChar.length);
```

Comment:
won't this break autocompleteMode in tributejs

Reply:
Good point. I think the idea of having a boolean become and integer for
artimatic purposes made me blitz it, but I didn't put back a proper
solution.

It likely needs a
```
autocompleteOffet = this.tribute.autocompleteMode ? info.mentionTriggerChar.length : 0;
```
and then use that offset value.